src/test/kotlin/uk/gov/justice/digital/hmpps/pecs/jpc/controller/HtmlControllerTest.kt

```diff
@@ -121,5 +127,20 @@ class HtmlControllerTest(@Autowired private val wac: WebApplicationContext) {
       .andExpect { status { is3xxRedirection() } }
 
     verify(moveService).findMoveByReferenceAndSupplier("REF1", defaultSupplierSerco)
+    verifyZeroInteractions(monitoringService)
+  }
+
+  @Test
+  internal fun `standard error page is shown when an unexpected exception occurs`() {
```

Comment:
Forcing an error in one of the known endpoints to ensure the error page is rendered.